CITATION.cff

```diff
@@ -0,0 +1,57 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+
+# Project information
+authors:
+  - name: "National Optical Astronomy Observatories"
+  - name: "The IRAF community"
+title: IRAF
+type: software
+abstract: "Image Reduction and Analysis Facility "
+repository-code: "https://github.com/iraf-community/iraf"
+url: "https://iraf-community.github.io"
+license-url: "https://github.com/iraf-community/iraf/blob/main/COPYRIGHT"
+
+# Version information
+version: "2.17"
+doi: "10.5281/zenodo.5816744"
+data-released: "2022-01-04"
+
+# Associated papers
+preferred-citation:
+  authors:
+    - family-names: Tody
+      given-names: Doug
+  title: "The IRAF Data Reduction and Analysis System"
+  type: article
+  issue: 627
+  journal: "Proc. SPIE Instrumentation in Astronomy VI"
+  year: 1986
+  pages: 733
+  doi: "10.1117/12.968154"
+  editors:
+    - family-name: Crawford
+      given-names: "D.L."
+  url: "https://www.spiedigitallibrary.org/conference-proceedings-of-spie/0627/1/The-Iraf-Data-Reduction-And-Analysis-System/10.1117/12.968154.short"
+references:
+  - authors:
+      - family-names: Tody
+        given-names: Doug
+    title: "IRAF in the Nineties"
+    type: article
+    conference: "Astronomical Data Analysis Software and Systems II"
+    volume: 52
+    journal: "A.S.P. Conference Series"
+    pages: 173
+    year: 1993
+    editors:
+      - family-name: Hanisch
+        given-names: "R. J."
+      - family-name:  Brissenden
+        given-names: "R. J. V."
+      - family-name: Barnes
+        given-names: Jeanette
+    url: "https://articles.adsabs.harvard.edu/pdf/1993ASPC...52..173T"
+
+  - repository: "https://ascl.net/9911.002"
```

Comment:
This should be rather the software's repository; see my comment in https://github.com/citation-file-format/citation-file-format/issues/436.